src/atomvm_pico_flash_provider.erl

```diff
@@ -57,8 +57,7 @@ init(State) ->
         {short_desc, "Convert an AtomVM packbeam file to uf2 and copy to a an rp2040 device"},
         {desc,
             "~n"
-            "Use this plugin to convert an AtomVM packbeam file to a rp2040 a uf2 file and copy to an rp2040 devices.~n"
-        }
+            "Use this plugin to convert an AtomVM packbeam file to a rp2040 a uf2 file and copy to an rp2040 devices.~n"}
```

Comment:
I’d keep the } on the next, line, but that’s just me.

But I think there was a typo even in the original.  Should it read “to a rp2040 uf2 file”. (remove the `a`)

Reply:
I accidentally ran erlfmt over the file, and missed undoing this unnecessary change.

Reply:
fixed.